src/main/java/org/hydev/mcpm/client/commands/entries/InfoController.java

```diff
@@ -0,0 +1,83 @@
+package org.hydev.mcpm.client.commands.entries;
+
+import com.google.common.collect.Streams;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.text.WordUtils;
+import org.hydev.mcpm.client.database.PluginTracker;
+
+import javax.annotation.Nullable;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Consumer;
+
+/**
+ * Controller for the info use case
+ *
+ * @author Azalea (https://github.com/hykilpikonna)
+ * @since 2022-11-22
+ */
+public record InfoController(PluginTracker tracker)
+{
+    private static final int LEN_LEFT = 12;
+    private static final int LEN_RIGHT = 60;
+
+    /**
+     * Format a kv pair if value isn't null, return empty string if value is null
+     *
+     * @param key Key
+     * @param rawValue Value
+     * @return Formatted pair
+     */
+    private static String formatPair(String key, @Nullable Object rawValue)
+    {
+        if (rawValue == null) return "";
+
+        String value;
+        if (rawValue instanceof Map<?,?> v) rawValue = v.keySet().stream().toList();
+        if (rawValue instanceof List<?> v) value = String.join(", ", v.stream().map(Object::toString).toList());
```

Comment:
Inline if body is one thing, but inline if body without a newline...

```suggestion
        if (rawValue instanceof List<?> v) {
            value = String.join(", ", v.stream().map(Object::toString).toList());
        }
```

Reply:
sorry 😭